dashboard/test/models/script_test.rb

```diff
@@ -194,6 +194,52 @@ def populate_cache_and_disconnect_db
     assert_equal 'Stage1', second.name
   end
 
+  test 'all fields can be set and then removed on reseed' do
```

Comment:
thanks for adding the test fixtures. now I'm having trouble seeing the difference between these test cases. are this test case and the corresponding fixtures now obsolete?

Reply:
I forgot to update the comments and variable names - I can do that - but one case covers only the stuff in `properties` and the other covers everything else (the fields that have their own db columns). I was going to do one test for everything initially, but I found it's impossible to set `hidden false` if `pilot_experiment` is set.

Reply:
ok, that makes sense -- "properties" is clear, but I didn't realize that "fields" meant database columns as opposed to properties, and I didn't notice that the longer list doesn't include the shorter list. comments and/or naming changes which indicate the difference between the tests and why there need to be two of them would be great, thanks!